keyrings/alt/file.py

```diff
@@ -37,7 +39,8 @@ class Encrypted(object):
     """
     PyCrypto-backed Encryption support
     """
-
+    scheme = 'PyCrypto [PBKDF2] AES256.CFB'
```

Comment:
Yeah, that would be great, but as of now, there're too many diverging crypto projects.
As always, the devil is in the details, and covering even the major ones is, ahem, ambitious...